src/core_plugins/interpreter/server/lib/get_plugin_stream.js

```diff
@@ -19,10 +19,12 @@
 
 import fs from 'fs';
 import ss from 'stream-stream';
-import { getPluginPaths } from './get_plugin_paths';
+import { getPluginPaths } from '@kbn/interpreter/server/get_plugin_paths';
 
 export const getPluginStream = type => {
-  const stream = ss();
+  const stream = ss({
+    separator: '\n',
```

Comment:
not using the separator sometimes produces broken js files, when joining multiple webpack bundles